pyartcd/pyartcd/pipelines/layered_products_scan_konflux.py

```
@@ -63,9 +63,8 @@ def check_params(self):
         """
 
         if not self.runtime.dry_run:
-            # TODO: Uncomment after logging can support stream builds
-            # if self.assembly != 'stream':
-            #     raise ValueError('non-stream assemblies are only allowed in dry-run mode')
+            if self.assembly != 'stream':
+                raise ValueError('non-stream assemblies are only allowed in dry-run mode')
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)